### ISSUE TITLE
how to do 2D Burgers hyperreduction?

### ISSUE THREAD

**Issue by gokhalen** [Pressio/pressio-tutorials#48]
I've been able to install pressio-tutorials and I'm able to run 2D SWE: hyper-reduced LSPG

https://pressio.github.io/pressio-tutorials/endtoend/swe_lspg_hypred_1.html

I've been able to install pressio-demoapps and I've been able to run the 2d Burgers' code

https://pressio.github.io/pressio-demoapps/burgers_2d_periodic.html

How would I go about doing hyper-reduction for 2D Burgers? Do I just need a new workflow file? Or, do I need to modify the driver files (wf_foms.py etc) residing in the following directory?

https://github.com/Pressio/pressio-tutorials/tree/develop/end-to-end-roms

Or, is there something else I need to do?

Thanks,

Nachiket

**Comment by fnrizzi** [comment]
yes, you have to add a custmozation for that inside https://github.com/Pressio/pressio-tutorials/tree/develop/end-to-end-roms 
look at the one existing for the SWE: https://github.com/Pressio/pressio-tutorials/blob/develop/end-to-end-roms/2d_swe_customizations/__init__.py

If you add: `2d_burgers_periodic_customizations` and put an __init__.py inside and edit it properly, then all you need is that I think.

**Comment by gokhalen** [comment]
I will try. I'm not sure about the pull request because I am not sure about my organization's policies towards open source contributions. I will try to check.
